index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Slider</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="slider">
+
+        <div class="slider__block">
+            <div class="slider__line">
+                <img src="images/1.jpg" alt="">
+                <img src="images/2.jpg" alt="">
+                <img src="images/3.jpg" alt="">
+            </div>
+        </div>
+
+        <button class="slider__button slider-prev"><</button>
```

Comment:
Було б круто додати `type` - це добра практика. Хто зна коли цей слайдер якимось чином опиниться в середині форми. Через те що хтось тег забув закрити, наприклад?
А ще: доступність. Зараз в нас скобочка в середині кнопки. Мені навіть цікаво, а що скаже скрін-рідер?
Може ось так зробимо?
`<` - може краще `&lt;`? Я завжди трохи нервую, коли бачу неекрановані спеціальні символи в HTML. 
```suggestion
        <button class="slider__button slider-prev" type="button" aria-label="Previous">&lt;</button>
```